hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/HmppsResourceServerConfiguration.kt

```diff
@@ -34,11 +37,15 @@ import uk.gov.justice.hmpps.kotlin.auth.dsl.ResourceServerConfigurationCustomize
 @EnableMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
 class HmppsResourceServerConfiguration {
   @ConditionalOnMissingFilterBean
+  @Order(LOWEST_PRECEDENCE)
```

Comment:
this seems odd.  If we've got `ConditionalOnMissingFilterBean` set anyway then there won't be any other filter beans, unless I'm missing something?